Sources/TurboIdentifiable.swift

```diff
@@ -0,0 +1,6 @@
+import UIKit
+
+/// A covenient way to identify view controllers.
+public protocol TurboIdentifiable : UIViewController {
+    static var viewControllerPathConfigIdentifier: String { get }
```

Comment:
I like that this is obvious exactly what it's used for. But I think it might be a tad too verbose. What about something like `pathConfigurationIdentifier` or even `turboIdentifier`?

The "view controller" part is already obvious (to me) if it exists on a `UIViewController` subclass.

Reply:
`pathConfigurationIdentifier` works for me 👍🏻

Reply:
What do you think about `public protocol PathConfigurationIdentifiable` and `static var pathConfigurationIdentifier` then?

Reply:
Renamed here: https://github.com/joemasilotti/TurboNavigator/pull/55/commits/13dad1a841617a307a145c9edf20b4a4fca985cd.